### ISSUE TITLE
Removing points from run

### ISSUE THREAD

**Issue by SpirinEgor** [wandb/client#810]
* Weights and Biases version: wandb, version 0.8.22
* Python version: Python 3.7.6
* Operating System: Linux

### Description

How can I manually remove some points from the run?

### What I Did

In [documentaion](https://docs.wandb.com/library/api) said, that I can modify `run.history`, but, as far as I know, `history` does not attribute, it's a method, so I can't set a new value to it.

**Comment by cvphelps** [comment]
Hi Egor, we don't really support updating run data after it's been logged, as this might affect reproducibility. Could you tell us a little more about your use case for updating run data after the experiment has finished?

**Comment by SpirinEgor** [comment]
I'm training nn and it failed, for example, at step 150, but I have a checkpoint for step 100. So, I want to resume training in the existing run, but in this case, my graphics will be scattered, which I don't want. Because of it, I'm wondering, how I can remove some points to continue working in same run instead of creating new.

**Comment by shawnlewis** [comment]
The best thing for now might be to create a whole new run. You can download the data from the previous run using our data API, and replay it, before logging new data. The timestamps will be wrong but the chart will be correct with step as the x-axis. https://docs.wandb.com/library/api

**Comment by SpirinEgor** [comment]
Okay, I got it.
Thanks!

**Comment by joplaete** [comment]
Hey, was this functionality added since? Culling a window of logs feels like useful functionality to us as well and we also have a usecase where we would use this post model collapse to resume from a past iteration checkpoint. Thanks

**Comment by alex-petrenko** [comment]
I also have a use case for this. In population-based training you sometimes want to kill unsuccessful models in the population and restart them from an older checkpoint with a different set of hyperparameters. Sometimes you need to do it multiple times trying multiple sets of parameters. As a result, WandB logs become a mess with different attempts writing over each other.

Deleting all metrics in a specified [x1, x2] window would've been very useful!

**Comment by vmorozov** [comment]
I train models with thousands targets. I understand that I can filter loss and errors Wandb Charts for specific targets. However, the project/run dashboards are frozen when I open the  web pages. I train models via Pytorch Tabular. So I can't setup the logger for specific metrics. An ability to delete specific metrics via Wandb Python API would be much appreciated.
